client/app/components/notifications/notification-body.html

```diff
@@ -0,0 +1,69 @@
+  <span class="drawer-pf-notification-message notification-message"
+        ng-click="customScope.markRead(notification, notificationGroup)"
+        tooltip-popup-delay="500" tooltip-placement="bottom" tooltip="{{notification.message | translate}}">
+    {{notification.message | translate}}
+  </span>
+  <div ng-if="!notification.data.inProgress" class="drawer-pf-notification-info" ng-click="customScope.markRead(notification, notificationGroup)">
+    <span class="date">{{notification.timeStamp | date:'MM/dd/yyyy'}}</span>
+    <span class="time">{{notification.timeStamp | date:'h:mm:ss a'}}</span>
+  </div>
+  <div ng-if="notification.data.inProgress" class="mini-progress clearfix">
+    <span class="time">{{notification.timeStamp | date:'h:mm:ss a'}}</span>
+    <div class="mini-progress-area">
+      <div class="progress">
+        <span class="progress-bar progress-bar-info"
+              ng-style="{width: notification.data.percentComplete + '%'}" tooltip="{{notification.data.percentComplete}} + {{'% Complete' | translate}}">
+        </span>
+      </div>
+    </div>
+  </div>
+</div>
+<div ng-if="drawerExpanded" class="container-fluid expanded-notification" ng-click="customScope.markNotificationRead(notification, notificationGroup)">
+  <div class="row">
+    <div ng-class="{'col-md-4': notificationGroup.notificationType == 'task'}" class="col-sm-6">
+      <span ng-if="notification.type" class="pull-left {{customScope.getNotficationStatusIconClass(notification)}}" ng-click="customScope.markRead(notification, notificationGroup)"></span>
+      <span class="drawer-pf-notification-message notification-message"
+            ng-click="customScope.markRead(notification, notificationGroup)"
+            tooltip-append-to-body="true" tooltip-popup-delay="500" tooltip-placement="bottom" tooltip="{{notification.message | translate}}">
+            {{notification.message | translate}}
+      </span>
+    </div>
+    <div class="col-md-4 col-sm-3" ng-if="notificationGroup.notificationType == 'task'">
+      <div class="drawer-pf-notification-info expanded-info" ng-click="customScope.markRead(notification, notificationGroup)">
+        <span class="info-title">Started: </span>
+        <span class="date">{{notification.data.startTime | date:'MM/dd/yyyy'}}</span>
+        <span class="time">{{notification.data.startTime | date:'h:mm:ss a'}}</span>
+      </div>
+      <div ng-if="notification.inProgress" class="progress">
+        <div class="progress-bar progress-bar-info"
+             ng-style="{width: notification.data.percentComplete + '%'}" tooltip="{{notification.data.percentComplete}} + {{'% Complete' | translate}}">
+        </div>
+      </div>
+    </div>
+    <div class="col-md-4 col-sm-3" ng-if="notificationGroup.notificationType == 'task'">
+      <div class="drawer-pf-notification-info" ng-click="customScope.markRead(notification, notificationGroup)">
+        <span class="info-title">Completed: </span>
```

Comment:
I'd say prefer translate attribute *if* you can sanely use it, but go for `{{""|translate}}` if you'd have to create elements just to be able to do the attribute :).

As for l10n, we simply currently don't support it. Would be nice though.. And IIRC we do have `moment` in there, with locale support, so maybe we could make a moment filter and set the appropriate moment locale when changing the active language .. and get at least a rudimentary l10n that way... OTOH, I'd say it's better to be consistently wrong than being inconsistently right, so.. if our date picker can't handle different formats reasonably, it'd be more suprising to have one format in edit and one in view than to force the US way in both..